app/src/app/components/search/search-results/search-results.component.scss

```diff
@@ -21,6 +21,16 @@
     flex-direction: row;
     align-items: center;
     border-bottom: 1px solid #eee;
+
+    .adf-search-filter-chip {
+      &.mat-chip {
+        background-color: #efefef;
```

Comment:
this should be coming from the css variable

Reply:
Changes